src/main.py

```diff
@@ -148,8 +148,6 @@ def main() -> NoReturn:
         print("")
     elif args.favorites:
```

Comment:
Maybe we should add an additional parameter `-j` for json output.

```suggestion
    elif args.favorites:
        favorites = scanner.get_favorites()
        if args.json:
            print(json.dumps(favorites, sort_keys=True))
        else:
            print("")
            ....
```

This way the user can specify whether he wants plain json or a formatted output.
Same for favorite ids and credentials.

Reply:
Logging could be disabled with:
```python
    args = parser.parse_args()

    if args.json:
        logging.disable(logging.CRITICAL)
```